website/static/js/contribManager.js

```diff
@@ -462,11 +423,39 @@ var ContributorsViewModel = function(contributors, adminContributors, user, isRe
 
 function ContribManager(selector, contributors, adminContributors, user, isRegistration, table, adminTable) {
     var self = this;
+    //shouter allows communication between ContribManager and ContribRemover, in particular which contributor needs to
+    // be removed is passed to ContribRemover
+    var contribShouter = new ko.subscribable();
```

Comment:
I tried that earlier, and again now.  I am getting timing issues.  The data will show after the page loads with the simple passed observable you are suggesting, but its not there when the page loads, and knockout breaks.